src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  let discount = 0;
+
+  switch (true) {
+    case days >= 3 && days < 7:
+      discount = 20;
+      break;
+    case days >= 7:
+      discount = 50;
```

Comment:
The condition `days >= 7` should be checked first since it's a more specific case and it will allow you to remove the `&& days < 7` part from the previous condition.